tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing.asciidoc

```diff
@@ -0,0 +1,247 @@
+:authors: Roberto Cortez
+:email: radcortez@yahoo.com
+:version-label!:
+:sectanchors:
+:doctype: book
+:license: Eclipse Foundation Specification License v1.0
+:source-highlighter: coderay
+:toc: left
+:toclevels: 4
+:sectnumlevels: 4
+ifdef::backend-pdf[]
+:pagenums:
+endif::[]
+:otel-spec-version: 1.8.0
+:otel-java-version: 1.9.0
+
+= Eclipse MicroProfile Telemetry Tracing
+
+== Introduction
+
+In cloud-native technology stacks, distributed and polyglot architectures are the norm. Distributed architectures
+introduce a variety of operational challenges including how to solve availability and performance issues quickly.
+These challenges have led to the rise of observability.
+
+Telemetry data is needed to power observability products. Traditionally, telemetry data has been provided by either
+open-source projects or commercial vendors. With a lack of standardization, the net result is the lack of data
+portability and the burden on the user to maintain the instrumentation.
+
+The https://opentelemetry.io[OpenTelemetry] project solves these problems by providing a single, vendor-agnostic
+solution.
+
+== Architecture
+
+https://opentelemetry.io[OpenTelemetry] is a set of APIs, SDKs, tooling and integrations that are designed for the
+creation and management of telemetry data such as traces, metrics, and logs.
+
+This specification defines the behaviors that allow MicroProfile applications to easily participate in an environment
+where distributed tracing is enabled via https://opentelemetry.io[OpenTelemetry] (a merger between
+https://opentracing.io[OpenTracing] and https://opencensus.io[OpenCensus]).
+
+This document and implementations *MUST* comply with the following OpenTelemetry {otel-spec-version} specifications:
+
+- https://github.com/open-telemetry/opentelemetry-specification/blob/v{otel-spec-version}/specification/overview.md[OpenTelemetry Overview]
+(except for Metrics)
+- https://github.com/open-telemetry/opentelemetry-specification/blob/v{otel-spec-version}/specification/trace/api.md[Tracing API]
+- https://github.com/open-telemetry/opentelemetry-specification/blob/v{otel-spec-version}/specification/baggage/api.md[Baggage API]
+- https://github.com/open-telemetry/opentelemetry-specification/tree/v{otel-spec-version}/specification/context[Context API]
+- https://github.com/open-telemetry/opentelemetry-specification/blob/v{otel-spec-version}/specification/resource/sdk.md[Resource SDK]
+
+IMPORTANT: The Metrics and Logging integrations of https://opentelemetry.io[OpenTelemetry] are out of scope of this
+specification. Implementations are free to provide support for both Metrics and Logging if desired.
+
```

Comment:
```suggestion
```suggestion
This specification supports the following three type of instrumentions: Automatic Instrumentation, Manual Instrumentation and Agent Instrumentation. 
```
```